src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/recordings/test_openshift_create_service_no_wait.yaml

```diff
@@ -0,0 +1,451 @@
+interactions:
```

Comment:
As a reminder, openshift commands are not well supported.

Reply:
Removed Openshift commands in hybrid profile test